gcloud/bigquery/table.py

```diff
@@ -50,6 +83,29 @@ def path(self):
         return '%s/tables/%s' % (self._dataset.path, self.name)
 
     @property
+    def schema(self):
+        """Table's schema.
+
+        :rtype: list of :class:`SchemaField`
+        :returns: fields describing the schema
+        """
+        return list(self._schema)
+
+    @schema.setter
+    def schema(self, value):
+        """Update table's schema
+
+        :type value: list of :class:`SchemaField`
+        :param value: fields describing the schema
+
+        :raises: TypeError if 'value' is not a sequence, or ValueError if
+                 any item in the sequence is not a SchemaField
+        """
+        if not all(isinstance(field, SchemaField) for field in value):
+            raise ValueError('Schema items must be fields')
+        self._schema = tuple(value)
```

Comment:
Only certain kinds of changes are allowed.   See: https://cloud.google.com/bigquery/docs/tables#updateschema